src/main/groovy/com/avioconsulting/mule/linter/rule/pom/PomExistsRule.groovy

```diff
@@ -0,0 +1,30 @@
+package com.avioconsulting.mule.linter.rule.pom
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.PomFile
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+
+class PomExistsRule extends Rule {
+
+    static final String RULE_ID = 'POM_EXISTS'
+    static final String RULE_NAME = 'pom.xml file exists'
+    static final String FILE_NOT_EXISTS = 'pom.xml does not exist.'
+
+    PomExistsRule() {
+        this.ruleId = RULE_ID
+        this.ruleName = RULE_NAME
+    }
+
+    @Override
+    List<RuleViolation> execute(Application app) {
+        List<RuleViolation> violations = []
+
+        PomFile pomFile = app.pomFile
+        if (pomFile == null || !pomFile.doesExist()) {
+            violations.add(new RuleViolation(this, app.applicationPath.toString(), 0, FILE_NOT_EXISTS))
```

Comment:
Shouldn't the location be the pomFile absolute path, not just the app directory?